apps/src/lab2/views/dialogs/ShareDialog.tsx

```diff
@@ -78,6 +87,41 @@ const AfeCareerTourBlock: React.FunctionComponent = () => {
   );
 };
 
+const SubmitButtonInfo: React.FunctionComponent<{
+  submissionStatus: string | undefined;
+  dispatch: AppDispatch;
+}> = ({submissionStatus, dispatch}) => {
+  if (!experiments.isEnabled(experiments.LAB2_SUBMIT_PROJECT)) {
+    return null;
+  }
+  if (submissionStatus === 'can_submit') {
```

Comment:
I'm guessing this maybe a TODO for when we actually hook this up to the API, but can we create a type for this?

Also very minor/personal preference, but can we camelize the data we send back from the server? I know we don't do this consistently, but probably ideal that we stick with one case for the server and one case for the frontend so the meaning is clear.

Reply:
> I'm guessing this maybe a TODO for when we actually hook this up to the API, but can we create a type for this?

Yes, TODO since defined in `shared_constants.rb`.

Reply:
> Also very minor/personal preference, but can we camelize the data we send back from the server? I know we don't do this consistently, but probably ideal that we stick with one case for the server and one case for the frontend so the meaning is clear.

Since these strings are defined in `shared_constants`, l think we'll just be able to refer to `ProjectSubmissionStatus`?

Reply:
Merged #61974 so can add type!